### ISSUE TITLE
[BUG] Installation failure

### ISSUE THREAD

**Issue by TheRarita** [p4535992/conditional-visibility-new#1]
It is not possible to install the module, when pasting the `https://raw.githubusercontent.com/p4535992/conditional-visibility-new/master/src/module.json` URL error occurs

![image](https://user-images.githubusercontent.com/51272447/156956923-b8e9821b-2719-4633-8b38-26e1cce0ce6e.png)

**Comment by TheRarita** [comment]
Makes sense, the URL I got is from the project description that is why I opened the issue :) thanks for the work you are doing

**Comment by DarKDinDoN** [comment]
So excited about this new version 😄
